test/MultiplicationTable.test.ts

```diff
@@ -13,4 +13,40 @@ describe('Multiplication Table', () => {
     //then
     expect(rendered).toBe('1*1=1')
   })
+  it('should return empty given (2, 1)', () => {
+    // given
+    const table = new MultiplicationTable()
+    const start = 2
+    const end = 1
+
+    //when
+    const rendered = table.render(start, end)
+
+    //then
+    expect(rendered).toBe('')
+  })
+  it('should return empty given (12, 16)', () => {
+    // given
+    const table = new MultiplicationTable()
+    const start = 12
+    const end = 16
+
+    //when
+    const rendered = table.render(start, end)
+
+    //then
+    expect(rendered).toBe('')
+  })
+  it('should return the three lines for (2, 4)', () => {
+    // given
+    const table = new MultiplicationTable()
+    const start = 2
+    const end = 4
+
+    //when
+    const rendered = table.render(start, end)
+    console.log(rendered);
+    //then
+    expect(rendered).toBe('2*2=4\n2*3=6  3*3=9\n2*4=8  3*4=12  4*4=16');
+  })
 })
```

Comment:
test case does not contain the AC2: The start number and the end number can be any integer in a range of 1 to 10 (inclusive).